linera-service/tests/end_to_end_tests.rs

```diff
@@ -1282,20 +1282,22 @@ async fn test_end_to_end_fungible() {
         .await;
     assert_eq!(value, Amount::from(1));
 
-    // Moving the money back to chain1 (but same owner -- it will be stuck there until we
-    // add "claim" operation to fungible).
-    let destination2 = format!(
+    // Claiming more money from chain1 to chain2.
+    let source = format!(
         "{{chainId: \"{}\", owner: {}}}",
         chain1,
         account_owner2.to_value()
     );
+    let destination = format!(
+        "{{chainId: \"{}\", owner: {}}}",
+        chain2,
+        account_owner2.to_value()
+    );
```

Comment:
That pattern of building the address repeats quite a lot and maybe deserve its own function to avoid repetition.

Reply:
Sure, this one I can do

Reply:
It was your code though :)

Reply:
Done. The best solution is actually to use `fungible::Account`.